packages/shorebird_cli/lib/src/commands/patch/ios_patcher.dart

```diff
@@ -304,17 +307,42 @@ class IosPatcher extends Patcher {
 
     final linkProgress = logger.progress('Linking AOT files');
     double? linkPercentage;
-    try {
-      final dumpDebugInfoDir = await aotTools.isLinkDebugInfoSupported()
-          ? Directory.systemTemp.createTempSync()
-          : null;
+    final dumpDebugInfoDir = await aotTools.isLinkDebugInfoSupported()
+        ? Directory.systemTemp.createTempSync()
+        : null;
 
-      if (dumpDebugInfoDir != null) {
-        logger.detail(
-          'Dumping link debug info to ${dumpDebugInfoDir.path}',
-        );
+    // The linking process can take a while and there might be cases where it
```

Comment:
Have you been able to reproduce the issue? We might want to hold off on merging anymore debugging changes until we can reproduce the issue. Also we just rolled Dart which may have resolved the issue for affected customers so we might want to double check and see if folks are still running into this with the latest version.

Reply:
> Have you been able to reproduce the issue?

I have not, all the things that I tested resulted in the link command failing, and the patch command would fail in the sequence, I didn't got to make it hang.

> We might want to hold off on merging anymore debugging changes until we can reproduce the issue.

We could hold off, but I see value in this PR nevertheless, before it, our debug files would be consolidated only on a linking success, this PR makes it happen when there is also a failing or an interrupt. So even if the issue is not reproducible, I still see value in the changes. Let me know if this makes sense